tools/test_containers_compose

```diff
@@ -20,7 +20,12 @@ wait_until() {
 
 setup_containers() {
   for retry in {2..0}; do sudo docker-compose build && break; echo "Remaining retries $retry"; done
-  sudo MOJO_CLIENT_DEBUG=1 docker-compose up -d
+  if ! sudo MOJO_CLIENT_DEBUG=1 docker-compose up -d; then
```

Comment:
Looks like you added the code I've suggested. Unfortunately, after reading and testing it myself again, I don't think it works (`exit_code` will be 0). This should work, however:

```suggestion
  sudo MOJO_CLIENT_DEBUG=1 docker-compose up -d || exit_code=$?
  if [[ -n $exit_code ]]; then
```

Reply:
Thanks, changed